### ISSUE TITLE
[Web] [Accessibility] [11yMAS] [MAS2.4.5] [Alternative way is not provided for the user to close the pop up.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab-2017#714]
**User Experience:**
User who rely on keyboard will not be able to close the dialog box without navigating inside the pop up. 

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools used: Narrator
**Attachments:**
[MAS2.4.5_Alternative way.zip](https://github.com/OfficeDev/script-lab/files/1457558/MAS2.4.5_Alternative.way.zip)

**Repro Steps:**
1. Open the URL: https://script-lab.azureedge.net/#/edit/excel.com
2. Navigate to “Delete” control and activate it.
3. Verify user is able to close the "Delete pop up" using Esc key or not  .

**Current Behavior:**
Delete pop up is not getting closed with Esc key. User should navigate inside the pop up and has to press enter on cancel to close the pop up. 

**Expected Behavior:**
Pop up should get closed using ESC key which is consistent behavior. More than one way should be provided for the user which will reduce the Burden on the user.

**Note:**
Same issue repro through out the application, User is not able to close the popup using ESC key.
**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2fb4445-fc01-4830-892a-4b9efeda098a}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=2.4.5
